packages/mdc-textfield/README.md

```diff
@@ -218,40 +242,47 @@ Mixin | Description
 `mdc-text-field-label-color($color)` | Customizes the text color of the label.
 `mdc-text-field-line-ripple-color($color)` | Customizes the color of the default line ripple of the text field.
 
-### `MDCTextField`
+## `MDCTextField` Properties and Methods
 
-See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+### Properties
```

Comment:
Don't need subheadings for Properties and Methods, just have one table after the other (the header cells should suffice)